create_framework/introduction.rst

```diff
@@ -38,8 +38,7 @@ you can use as is or as a start for your very own. It will start with a simple
 framework and more features will be added with time. Eventually, you will have
 a fully-featured full-stack web framework.
 
-And of course, each step will be the occasion to learn more about some of the
-Symfony Components.
+Each step will be the occasion to learn more about some of the Symfony Components.
```

Comment:
We are using "Symfony components" (2/3) and "Symfony Components" (1/3) throughout the documentation.
Which one should we use?

I would propose a PR after this one is merged.